index.html

```diff
@@ -531,85 +531,78 @@ <h3>Proofs</h3>
         </p>
 
         <pre class="example highlight" title="A simple JSON data document">
-  {
-    "title": "Hello world!"
-  };
+{
+  "title": "Hello world!"
+};
         </pre>
 
         <p>
-by adding the parameters outlined in this section:
+The following proof secures the document above using the `jcs-eddsa-2022`
+cryptography suite [[?DI-EDDSA]], which produces a verifiable digital proof by
+transforming the input data using the JSON Canonicalization Scheme (JCS)
+[[?RFC8785]] and then digitally signing it using an Edwards Digital Signature
+Algorithm (EdDSA).
         </p>
 
         <pre class="example highlight"
         style="overflow-x: auto; white-space: pre-wrap; word-wrap: break-word;"
         title="A simple signed JSON data document">
-  {
-    "title": "Hello world!",
-    "proof": {
-      "type": "DataIntegrityProof",
-      "cryptosuite": "example-signature-2022",
-      "created": "2020-11-05T19:23:24Z",
-      "verificationMethod": "https://di.example/issuer#z6MkjLrk3gKS2nnkeWcmcxi
-        ZPGskmesDpuwRBorgHxUXfxnG",
-      "proofPurpose": "assertionMethod",
-      "proofValue": "zQeVbY4oey5q2M3XKaxup3tmzN4DRFTLVqpLMweBrSxMY2xHX5XTYV8nQA
-        pmEcqaqA3Q1gVHMrXFkXJeV6doDwLWx"
-    }
+{
+  "title": "Hello world!",
+  "proof": {
+    "type": "DataIntegrityProof",
+    "cryptosuite": "jcs-eddsa-2022",
+    "created": "2023-03-05T19:23:24Z",
+    "verificationMethod": "https://di.example/issuer#z6MkjLrk3gKS2nnkeWcmcxi
+      ZPGskmesDpuwRBorgHxUXfxnG",
+    "proofPurpose": "assertionMethod",
+    "proofValue": "zQeVbY4oey5q2M3XKaxup3tmzN4DRFTLVqpLMweBrSxMY2xHX5XTYV8nQA
+      pmEcqaqA3Q1gVHMrXFkXJeV6doDwLWx"
```

Comment:
This makes it look like the inline whitespace in the `verificationMethod` and `proofValue` will be ignored by tools that make use of these values. I'm pretty sure that's *not* the reality, so I think the above should become the following:
```suggestion
    "verificationMethod": "https://di.example/issuer#z6MkjLrk3gKS2nnkeWcmcxiZPGskmesDpuwRBorgHxUXfxnG",
    "proofPurpose": "assertionMethod",
    "proofValue": "zQeVbY4oey5q2M3XKaxup3tmzN4DRFTLVqpLMweBrSxMY2xHX5XTYV8nQApmEcqaqA3Q1gVHMrXFkXJeV6doDwLWx"
```
It *might* make sense to put `&nbsp;` between the keys and the values, so that the wraps are guaranteed to be in the long strings....

Reply:
I attempted a fix in 3b1631f38ad34248a622f88ba44408f902c9aa86. The upside is that people can now just copy-paste the examples AND the line wrapping happens automatically on small screens. The downside is that the examples are less readable w/o the indentation. 

If we want to do better here, we will need to write a respec plugin that automatically wraps and indents long lines in &lt;pre> HTML tags, which would be a non-trivial undertaking if we wanted highlighting to work as well. CSS is hard.

Reply:
`wbr` works for me